src/analysis/analytic/basic.lean

```diff
@@ -82,8 +82,9 @@ namespace formal_multilinear_series
 
 variables (p : formal_multilinear_series 𝕜 E F) {r : ℝ≥0}
 
-/-- The radius of a formal multilinear series is the largest `r` such that the sum `Σ pₙ yⁿ`
-converges for all `∥y∥ < r`. -/
+/-- The radius of a formal multilinear series is the largest `r` such that the sum `Σ ∥pₙ∥ ∥y∥ⁿ`
+converges for all `∥y∥ < r`. This implies that `Σ pₙ yⁿ` converges for all `∥y∥ < r`, but these
+definitions are *not* equivalent in general. -/
```

Comment:
do you have an example showing that these definitions are not equivalent? I agree there is no straightforward implication, but for instance over the complexes they turn out to be equivalent, so I'd just remove the end of the sentence, starting from `but these ...`.

Reply:
Yes I think I have one : take `E = F = ℝ³` with the cross product. Then `Σ yⁿ` converges for all `y`, since `y × y = 0`, but the series has radius one.

Reply:
Yes, that's a good one!